src/Form43Checker.py

```diff
@@ -89,32 +89,19 @@ def __getForm(self, file):
         else:
             return "invalid"
 
+    def __listInvalidFiles(self):
+        for file in self.__organizer.findFiles("", "*.es[pm]"):
```

Comment:
I wouldn't know shit about this, but this doesn't call findFiles for each yield right?

Reply:
No, this call findFiles only once to get a generator out of it.